galadriel/cli.py

```diff
@@ -241,57 +238,63 @@ def import_wallet(private_key: str, path: str):
     if private_key and path:
         raise click.ClickException("Please provide only one of --private-key or --path")
 
-    # FIXME Disable this check for now
-    # Check if the .agents.env file exists
-    # if not os.path.exists(".agents.env"):
-    #    raise click.ClickException(
-    #        "No .agents.env file found in current directory. Please run this command under your project directory."
-    #    )
+    try:
+        if private_key:
+            # Import from private key
+            keypair_bytes = bytes(json.loads(private_key))
+            keypair = Keypair.from_bytes(keypair_bytes)
+            pubkey = str(keypair.pubkey())
+
+            # Save to default path
+            os.makedirs(os.path.dirname(DEFAULT_SOLANA_KEY_PATH), exist_ok=True)
+            with open(DEFAULT_SOLANA_KEY_PATH, "w") as f:
+                f.write(private_key)
+
+            click.echo(f"Successfully imported wallet {pubkey} to {DEFAULT_SOLANA_KEY_PATH}")
+        else:
+            # Import from file
+            with open(path, "r") as f:
+                keypair_json = f.read()
 
-    if private_key:
-        # Check if the private key is a valid json
-        try:
-            json.loads(private_key)
-        except json.JSONDecodeError:
-            raise click.ClickException("Invalid private key! Please provide a valid JSON array")
-        # Save the private key to the default path
-        os.makedirs(os.path.dirname(DEFAULT_SOLANA_KEY_PATH), exist_ok=True)
-        with open(DEFAULT_SOLANA_KEY_PATH, "w", encoding="utf-8") as file:
-            file.write(private_key)
-        _update_agent_env_file({"SOLANA_KEY_PATH": DEFAULT_SOLANA_KEY_PATH})
```

Comment:
removing this will cause issues as agents rely on the .agent.env to get the wallet key path